natalie.swift

```diff
@@ -555,6 +555,125 @@ extension XMLElement: Printable {
     }
 }
 
+
+enum OS: String, Printable{
+    case iOS = "iOS"
+    case OSX = "OSX"
+    
+    static func fromString(string: String) -> OS? {
+        for os in OS.allValues {
+            if  NSString(string: os.rawValue).caseInsensitiveCompare(string) == NSComparisonResult.OrderedSame {
+                return os
+            }
+        }
+        return nil
+    }
+    
+    static func fromTargetRuntime(targetRuntime: String) -> OS? {
+        for os in OS.allValues {
+            if  os.targetRuntime == targetRuntime {
+                return os
+            }
+        }
+        return nil
+    }
+    
+    static let allValues = [iOS, OSX]
+    
+    var description: String {return self.rawValue}
+    
+    var framework: String {
+        switch self {
+        case iOS: return "UIKit"
+        case OSX: return "Cocoa"
+        }
+    }
+    
+    var targetRuntime: String {
+        switch self {
+        case iOS: return "iOS.CocoaTouch"
+        case OSX: return "MacOSX.Cocoa"
+        }
+    }
+    
+    var storyboardType: String {
+        switch self {
+        case iOS: return "UIStoryboard"
+        case OSX: return "NSStoryboard"
+        }
+    }
+    
+    var storyboardTypeUnwrap: String {
```

Comment:
for iOS : init(name name: String,
   bundle storyboardBundleOrNil: NSBundle?) -> UIStoryboard

for MacOSX : convenience init**?**(name name: String,
           bundle storyboardBundleOrNil: NSBundle?)
This init() must be unwrapped